packages/atlas/src/components/_ypp/YppAuthorizationStepModal/YppAuthorizationStepModal.tsx

```diff
@@ -0,0 +1,140 @@
+import appScreenshot from '@/assets/images/ypp-authorization/app-screenshot.webp'
+import { Text } from '@/components/Text'
+import { Button } from '@/components/_buttons/Button'
+import { DialogModal, DialogModalProps } from '@/components/_overlays/DialogModal'
+
+import { AdditionalSubtitle, Content, Img, StyledSvgAtlasLogoShort } from './YppAuthorizationStepModal.styles'
+
+import { DetailsFormDialog } from '../DetailsFormDialog'
+import { RequirementsDialog } from '../RequirementsDialog'
+import { SelectChannelDialog } from '../SelectChannelDialog'
+import { TermsAndConditionsDialog } from '../TermsAndConditionsDialog'
+
+type Step = 'select-channel' | 'requirements' | 'details' | 'terms-and-conditions' | 'summary' | 'connect-with-yt'
+
+export type YppAuthorizationStepModalProps = {
+  onPrimaryClick?: () => void
+  onBackClick?: () => void
+  onCancelClick?: () => void
+  step: Step
+  channels: BasicChannelFieldsFragment[]
+} & Pick<DialogModalProps, 'show'>
+
+export const YppAuthorizationStepModal: FC<YppAuthorizationStepModalProps> = ({
+  show,
+  onBackClick,
+  onCancelClick,
+  channels,
+  step,
+}) => {
+  const isSummary = step === 'summary'
+  const isConnectWithYt = step === 'connect-with-yt'
+  const authorizationStep = useMemo(() => {
+    switch (step) {
+      case 'select-channel':
+        return {
+          title: 'Select channel',
+          description: 'Select the Atlas channel you want your YouTube channel to be connected with.',
+          primaryButton: { text: 'Select channel' },
+          component: <SelectChannelDialog channels={channels} />,
+        }
+      case 'requirements':
+        return {
+          title: 'Requirements',
+          description:
+            'Before you can apply to the program, make sure both your Atlas and YouTube channels meet the below conditions.',
+          primaryButton: {
+            text: 'Authorize with YouTube',
+          },
+          component: <RequirementsDialog />,
+        }
+      case 'details':
+        return {
+          title: 'Details',
+          description: 'We need your email address to send you payment information. No spam or marketing materials.',
+          primaryButton: {
+            text: 'Continue',
+          },
+          component: <DetailsFormDialog />,
+        }
+      case 'terms-and-conditions':
+        return {
+          title: 'Terms & conditions',
+          description:
+            'Once automatic YouTube videos sync is available, in order for it to work, your Atlas channel [NEEDS TO DO WHAT?]. This is purely a technical measure and does not affect ownership and rights to the content uploaded to you Atlas channel.',
+          primaryButton: { text: 'Accept terms & sign' },
+          additionalSubtitleNode: (
+            <AdditionalSubtitle>
+              <Text variant="h400" as="span">
+                Automatic YouTube sync
+              </Text>{' '}
+              <Text variant="t100" as="span" color="colorText">
```

Comment:
This should be `textMuted`
![image](https://user-images.githubusercontent.com/51168865/190170530-42ae56a4-a89d-4f28-a626-34ccba3a57ce.png)